compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -4173,9 +4173,9 @@ class Typer(@constructorOnly nestingLevel: Int = 0) extends Namer
           readapt(tree.appliedToNone) // insert () to primary constructors
         else
           errorTree(tree, em"Missing arguments for $methodStr")
-      case _ => tryInsertApplyOrImplicit(tree, pt, locked) {
-        errorTree(tree, MethodDoesNotTakeParameters(tree))
-      }
+      case _ =>
+        tryInsertApplyOrImplicit(tree, pt, locked):
+          errorTree(tree, MethodDoesNotTakeParameters(tree))
```

Comment:
This delta was due to a println. Philosophically, `tryInsert` is the important part, and putting it right of the arrow makes it less visible, and unduly highlights `errorTree`. The edit with new syntax is clearer.

Usual idiom where "thing to the right" is less important for understanding because it doesn't carry much information:
```
def f(x: X) = x match
  case _: Y =>
```